docs/get_started/quickstart.rst

```diff
@@ -8,32 +8,359 @@ Quick Start
+              METRIC   Round 9, aggregator: train <openfl.interface.aggregation_functions.weighted_average.WeightedAverage object at              aggregator.py:838
+                       0x7f329a98bee0> cross_entropy:    0.039476
+   [15:36:39] METRIC   Round 9, aggregator: aggregated_model_validation <openfl.interface.aggregation_functions.weighted_average.WeightedAverage  aggregator.py:838
+                       object at 0x7f329a98bee0> accuracy:   0.982298
+              METRIC   Round 9: saved the best model with score 0.982298                                                                          aggregator.py:854
+              METRIC   Round 9, aggregator: locally_tuned_model_validation                                                                        aggregator.py:838
+                       <openfl.interface.aggregation_functions.weighted_average.WeightedAverage object at 0x7f329a98bee0> accuracy:
+                       0.978598
+              INFO     Saving round 10 model...                                                                                                   aggregator.py:890
+              INFO     Experiment Completed. Cleaning up...                                                                                       aggregator.py:895
+   [15:36:39] INFO     Waiting for tasks...                                                                                                     collaborator.py:178
+              INFO     Sending signal to collaborator collaborator1 to shutdown...                                                                aggregator.py:283
+              INFO     End of Federation reached. Exiting...                                                                                    collaborator.py:150
+   
+    ✔ OK
+   [15:36:46] INFO     Waiting for tasks...                                                                                                     collaborator.py:178
+   [15:36:46] INFO     Sending signal to collaborator collaborator2 to shutdown...                                                                aggregator.py:283
+              INFO     End of Federation reached. Exiting...                                                                                    collaborator.py:150
+   
+    ✔ OK
+   
+   Congratulations! You've run your first federation with OpenFL
+
+***************************
+Working with your own model
+***************************
+
+Now that you've run your first federation, let's see how to use replace the model used in the federation. After copying in the text above, you should be in the :code:`my_workspace` directory. Every workspace has a :code:`src` directory that contains the Task Runner, an OpenFL interface that defines the deep learning model, as well as the training and validation functions that will run on that model. In this case, the Task Runner is defined in :code:`src/taskrunner.py`. After opening it you'll see the following:

```

Comment:
minor typo: "...let's see how **to use replace** the model used in the federation..."